infrastructure/lambda/template.yaml

```diff
@@ -632,21 +632,6 @@ Resources:
         BurstLimit: 100 # requests the API can handle concurrently
         RateLimit: 50 # allowed requests per second
 
-  PrivateAddressApiUsagePlan:
-    Type: AWS::ApiGateway::UsagePlan
-    DependsOn:
-      - PrivateAddressApiStage
-    Properties:
-      ApiStages:
-        - ApiId: !Ref PrivateAddressApi
-          Stage: !Ref Environment
-      Quota:
-        Limit: 500000
-        Period: DAY
-      Throttle:
-        BurstLimit: 100 # requests the API can handle concurrently
-        RateLimit: 50 # allowed requests per second
-
```

Comment:
I think we should retain this usage plan as it protects our API from being spammed by calls to the frontend